WebAppTokenStore/WebAppTokenStore/Controllers/HomeController.cs

```diff
@@ -10,30 +10,30 @@
 using Dropbox.Api;
 using Newtonsoft.Json;
 
-namespace WebAppTokenVault.Controllers
+namespace WebAppTokenStore.Controllers
 {
     public class HomeController : Controller
     {
-        const string TokenVaultResource = "https://tokenvault.azure.net";
+        const string TokenStoreResource = "https://tokenstore.azure.net";
```

Comment:
I haven't made this change to add this audience to the AAD app yet.

Reply:
added and finished testing